modules/ossm-mixer-policy-1x.adoc

```diff
@@ -10,6 +10,7 @@ In previous versions of {ProductName}, Mixer’s policy enforcement was enabled
 .Prerequisites
 * Access to the {product-title} Command-line Interface (CLI) also known as `oc`.
 
+NOTE: `istio-system` is the default namespace the Service Mesh Control Plane (SMCP) installs to. You can change `istio-system` to any other name during the installation.
```

Comment:
There is no default namespace for the SMCP, customers create the namespace and then create the SMCP in that namespace.  A better note might be "The examples use `istio-system` as the control plane namespace.  Replace this value with the namespace where you deployed the SMCP."  Or even better, change the sample commands to show `istio-system` in angle brackets to indicate that it's a user configurable value.   \<istio-system>